Unable to install on M1 Macbook

<!--

  ! PLEASE HELP US HELP YOU !

  Bugs are fixed faster if you include:
  - a repro repository to inspect the code
  - an url to see the problem live (if possible)

  Pro tip: create a reproducible demo of the bug with https://new.docusaurus.io

-->

## 🐛 Bug Report

Unable to install `@docusaurus/plugin-ideal-image@2.0.0-beta.0` on my M1 MacBook. 

This issue has been addressed in https://github.com/lovell/sharp/issues/2460. We can fix this by updating the dependency `sharp` from `^0.27.1` to `^0.28.0`.

### Have you read the [Contributing Guidelines on issues] 

yes

## To Reproduce

If possible, use https://new.docusaurus.io to create a CodeSandbox reproducible demo of the bug.
 
1. get a M1 Macbook
2. run `npm install @docusaurus/plugin-ideal-image`

## Expected behavior
 
Not error 

## Actual Behavior
 
```
../src/common.cc:24:10: fatal error: 'vips/vips8' file not found
```


<details>

```
 /tmp/test3  npm install @docusaurus/plugin-ideal-image  
⸨      ░░░░░░░░░░░░⸩ ⠋ idealTree:xml2js: sill placeDep ROOT xml2js@0.4.23 OK for: parse-bmfont-xml@1.1.4 want: ^0.4.5
⸨       ░░░░░░░░░░░⸩ ⠧ idealTree:ajv-keywords: sill fetch manifest uri-js@^4.2.2
npm ERR! code 1
npm ERR! path /private/tmp/test3/node_modules/sharp
npm ERR! command failed
npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
npm ERR! CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o
npm ERR!   LIBTOOL-STATIC Release/nothing.a
npm ERR!   TOUCH Release/obj.target/libvips-cpp.stamp
npm ERR!   CXX(target) Release/obj.target/sharp/src/common.o
npm ERR! info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.5/libvips-8.10.5-darwin-arm64v8.tar.br
npm ERR! ERR! sharp Prebuilt libvips 8.10.5 binaries are not yet available for darwin-arm64v8
npm ERR! info sharp Attempting to build from source via node-gyp but this may fail due to the above error
npm ERR! info sharp Please see https://sharp.pixelplumbing.com/install for required dependencies
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@7.1.2
npm ERR! gyp info using node@16.1.0 | darwin | arm64
npm ERR! gyp info find Python using Python version 3.8.2 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
npm ERR! (node:52342) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
npm ERR! (Use `node --trace-deprecation ...` to show where the warning was created)
npm ERR! gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/private/tmp/test3/node_modules/sharp/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/ocavue/Library/Caches/node-gyp/16.1.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/ocavue/Library/Caches/node-gyp/16.1.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/ocavue/Library/Caches/node-gyp/16.1.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/private/tmp/test3/node_modules/sharp',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! warning: /Library/Developer/CommandLineTools/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
npm ERR! ../src/common.cc:24:10: fatal error: 'vips/vips8' file not found
npm ERR! #include <vips/vips8>
npm ERR!          ^~~~~~~~~~~~
npm ERR! 1 error generated.
npm ERR! make: *** [Release/obj.target/sharp/src/common.o] Error 1
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:365:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! gyp ERR! System Darwin 20.3.0
npm ERR! gyp ERR! command "/opt/homebrew/Cellar/node/16.1.0/bin/node" "/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /private/tmp/test3/node_modules/sharp
npm ERR! gyp ERR! node -v v16.1.0
npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ocavue/.npm/_logs/2021-05-18T03_33_06_510Z-debug.log

```

</details>

 

## Your Environment
 
- Public source code: N/A
- Public site url: N/A
- Docusaurus version used: 2.0.0-beta.0
- Environment name and version (e.g. Chrome 78.0.3904.108, Node.js 10.17.0): node v16.1
- Operating system and version (desktop or mobile): mac os v11.2 (M1 Macbook)

## Reproducible Demo
 
N/A

--- a/packages/docusaurus-plugin-ideal-image/package.json
+++ b/packages/docusaurus-plugin-ideal-image/package.json
@@ -26,7 +26,7 @@
     "@docusaurus/types": "2.0.0-beta.0",
     "@endiliey/react-ideal-image": "^0.0.11",
     "react-waypoint": "^9.0.2",
-    "sharp": "^0.27.1",
+    "sharp": "^0.28.2",
     "tslib": "^2.1.0",
     "webpack": "^5.28.0"
   },

--- a/packages/lqip-loader/package.json
+++ b/packages/lqip-loader/package.json
@@ -16,7 +16,7 @@
     "file-loader": "^6.2.0",
     "lodash": "^4.17.20",
     "node-vibrant": "^3.1.5",
-    "sharp": "^0.27.1"
+    "sharp": "^0.28.2"
   },
   "engines": {
     "node": ">=12.13.0"

